profiling/src/pcntl.rs

```diff
@@ -94,6 +95,8 @@ fn stop_and_forget_profiling(maybe_profiler: &mut Option<Profiler>) {
     swap(&mut *maybe_profiler, &mut old_profiler);
     forget(old_profiler);
 
+    allocation_profiling_rshutdown();
```

Comment:
Is this really safe to call? Maybe we need to go through it line-by-line together in Zoom.

Reply:
I scheduled an appointment 👍